CMakeLists.txt

```diff
@@ -886,11 +1067,11 @@ if ( NOT DISABLE_SPICY )
         # seems this does not work reliable (due incomplete support for object
         # libraries in CMake?).
         list(TRANSFORM _spicy_rt_libs APPEND "-objects")
-        target_link_libraries(zeek ${_spicy_rt_libs})
+        target_link_libraries(${spicyZeekTarget} PUBLIC ${_spicy_rt_libs})
 
         # Make sure we build targets of spicy-plugin after the `spicy` target.
-        add_dependencies(plugin-Zeek-Spicy spicy)
-        add_dependencies(spicyz spicy)
+        #add_dependencies(plugin-Zeek-Spicy spicy)
```

Comment:
Are these needed anymore?

Reply:
Probably not. Once I got it all working again, I'll make a clean-up pass.